plugins/GSdx/Window/GSLinuxDialog.cpp

```diff
@@ -304,7 +302,7 @@ void populate_hw_table(GtkWidget* hw_table)
 
 	s_table_line = 0;
 	InsertWidgetInTable(hw_table , paltex_check  , acc_date_check);
-	InsertWidgetInTable(hw_table , large_fb_check, hack_enable_check);
+	InsertWidgetInTable(hw_table , hack_enable_check);
```

Comment:
I forgot how linux gui looks so this might need adjusting. A screenshot will be nice to see.